### ISSUE TITLE
Add delayed tibial epiphyseal ossification

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10525]
**Preferred term label:**
Delayed tibial epiphyseal ossification

**Synonyms**
Delayed ossification of the tibial epiphysis - Delayed ossification tibial epiphyses

**Definition (free text, please give PubMed ID)**
Developmental delay of ossification of the tibia. https://www.sciencedirect.com/science/article/pii/B9781416063933000312

**Parent term (use hpo.jax.org/app)**
Delayed epiphyseal ossification HP:0002663

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Achondrogenesis type 2 (omim200610), for example PMID24144387
Congenital syphilis (ORPHA:499009) PMID: 9559711

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000867
- label: Delayed tibial epiphyseal ossification
- parents: [Abnormality of tibial epiphyses (HP:0006508)](https://hpo.jax.org/app/browse/term/HP:0006508)
- pmid: PMID:24144387|PMID:9559711
- def: Developmental delay of ossification of an epiphysis of the tibia.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10525
- synonyms:
